content/reference/admin/private_locations/configuration/gcp/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "GCP Load Generators"
+description: "Load Generators on your private GCP account"
+lead: "Private Locations on your GCP account"
+date: 2023-10-02T15:29:00+00:00
+lastmod: 2023-10-02T15:29:00+00:00
+weight: 22054
+---
+
+## GCP Virtual Machines
+GCP private locations require the control plane to have GCP access rights configured in order to instantiate virtual machines.
+
+### Service account
+Access rights can be set through a service account associated to your control plane.
+
+Check GCP and Gatling documentation pages for more details:
+* [GCP Service account](https://cloud.google.com/iam/docs/service-account-overview)
+* [Gatling installation guide]({{< ref "../../installation/gcp/#service-account" >}})
+
+### Control plane configuration file
+
+```bash
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane token with an environment variable
+  token = ${?CONTROL_PLANE_TOKEN}
+  # Control plane token with a system property
+  token = $?control.plane.token
+  # Control plane description (optional)
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
+    {
+      # Private location ID, must be prefixed by prl_, only consist of numbers 0-9, 
+      # lowercase letters a-z, and underscores, with a max length of 30 characters
+      id = "prl_private_location_example"
+      # Private location description (optional)
+      description = "Private Location on GCP"
+      # Private location type
+      type = "gcp"
+      # GCP location name, as listed by GCP CLI:
+      # gcloud compute zones list
+      zone = "europe-west3-a"
+      machine {
+        # Virtual machine type, as listed by GCP CLI:
+        # gcloud compute machine-types list --filter="zone:( europe-west3-a )"
+        type = "e2-micro"
+        # Certified image configuration
+        image {
+          type = "certified"
+          java = "latest" # Possible values : 8, 11, 17 or latest
+        }
+        # Storage configuration
+        disk {
+          # Disk size in Gb
+          sizeGb = 20
+        }
+      }
+      # GCP project id as returned by GCP CLI:
+      # gcloud projects list
+      project = "<MyProject ID>"
+      # Java configuration (following configuration properties are optional)
+      # System properties (optional)
+      system-properties {
+        "java.net.preferIPv6Addresses" = "true"
+      }
+      # Overwrite JAVA_HOME definition (optional)
+      java-home = "/usr/lib/jvm/zulu17"
```

Comment:
I think there is no more suffix in our official images.
```suggestion
      java-home = "/usr/lib/jvm/zulu"
```

@notdryft, you can confirm ?

Reply:
Only with java 21 not supported yet